internal/receiver/lightprometheusreceiver/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+type: lightprometheus
+
+status:
+  class: receiver
+  stability: development
```

Comment:
should we add pipelines and distributions to fill the gaps in the generated table?
```suggestion
  stability: development
  pipelines: [metrics]
  distributions: [splunk-otel-collector]

```

Reply:
This approach will likely fail as the generator is not aware of a mapping from `splunk-otel-collector` to a URL. I am happy to go and open a request for enhancement towards adding that mapping.

Reply:
Ok, can we add just pipelines for now?

Reply:
oh yes that works.